verification/utils/definitions/definitions.gobra

```diff
@@ -18,28 +18,28 @@ package definitions
 
 const HalfPerm perm = 1/2
 const (
-	ReadL00 perm = 1/(2 << iota)
-	ReadL0
-	ReadL1
-	ReadL2
-	ReadL3
-	ReadL4
-	ReadL5
-	ReadL6
-	ReadL7
-	ReadL8
-	ReadL9
-	ReadL10
-	ReadL11
-	ReadL12
-	ReadL13
-	ReadL14
-	ReadL15
-	ReadL16
-	ReadL17
-	ReadL18
-	ReadL19
-	ReadL20
+	R00 perm = 1/(2 << iota)
+	R0
+	R1
+	R2
+	R3
+	R4
+	R5
+	R6
+	R7
+	R8
+	R9
+	R10
+	R11
+	R12
+	R13
+	R14
+	R15
+	R16
+	R17
+	R18
+	R19
+	R20
```

Comment:
I think Markus used 21 at some point and extended this file. Could we add some more definitions just to have breathing space?